### ISSUE TITLE
LicenseInfo for specific licenses in catalog tests should be extracted

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse#1978]
## Problem

Currently, several tests declare various license_info constants. This leads to duplication.

## Description

We should extract the license constants used in tests to a separate file, and import them from there. What is the best place to put those constants, @WordPress/openverse-catalog ? I've added `ticket work required` label to this issue to clarify the file location.

## Alternatives

We could also define the constants in the `common/licenses`, but there are a lot of possible licenses, so I think defining the constants that are specifically used in tests would be simpler.

## Additional context

Extracted from a PR comment: https://github.com/WordPress/openverse/pull/1898#discussion_r1181719389

**Comment by krysal** [comment]
I believe the `dags/common/licenses/constants.py` file should be a good place for those constants.

**Comment by stacimc** [comment]
If these are only used in tests, and especially if we're only defining a subset of licenses as constants, it might be preferable to keep them in the `catalog/tests/` directory.

**Comment by obulat** [comment]
> If these are only used in tests, and especially if we're only defining a subset of licenses as constants, it might be preferable to keep them in the `catalog/tests/` directory.

Yes, these are used only in tests, and there's just a subset of the most popular licenses. That is why I didn't think `dags/common/licenses/constants.py` was the best place for it. Do you think something like `catalog/tests/licenses.py` would be a good place, @stacimc? Or should we create a folder for them, like `catalog/tests/constants/licenses.py`?

**Comment by stacimc** [comment]
Agreed! There's an existing `catalog/tests/utilities` directory for literally testing things that are utilities, which could maybe cause confusion. I think I like `catalog/tests/testing_utilities/licenses.py` :)

**Comment by AetherUnbound** [comment]
🤦🏼‍♀️ Should have checked first haha, that's perfect!

**Comment by sarayourfriend** [comment]
I've updated the issue description to reflect @stacimc's suggestion and the various positive emoji reactions on it and removed the ticket work required. I've added "help wanted" as it seems straightforward enough for a community member to implement.
